### ISSUE TITLE
Copying certain properties using “copy”

### ISSUE THREAD

**Issue by lnaie** [SteveSanderson/knockout.mapping#112]
I'm still seeing observables for all those excluded properties set with "copy".
Is that to be expected? 
If so, what shall we use to not track some properties, but still have them copied over?

Thx

**Comment by sagacity** [comment]
No, `copy` should be okay. Can you put up a reproduction scenario on jsfiddle?
